Cargo.toml

```diff
@@ -7,6 +7,7 @@ rust-version = "1.64.0"
 members = [
     "rrule",
     "rrule-debugger",
+    "rrule-wasm"
```

Comment:
Does it need to be a separate crate? Why not add this to the `rrule` crate behind a `wasm` feature flag?

Reply:
I think the goal could be to use a feature flag but at the moment would be good to keep separated. Probably it's going to be necessary 'duplicate' the rrule models and change to make it compatible with [wasm_bindgen].

Reply:
Hmm, I still don't see why it needs its own crate. I believe it is better for this to be in a separate module in the `rrule` crate called `wasm` where all the wasm related bindings and duplicate models can exist. What do you think?

Reply:
I agree with you.